jenkins/release.jenkinsFile

```diff
@@ -1,35 +1,156 @@
+            regexpFilterText: ('$isDraft $action'),
+            regexpFilterExpression: ('^true created$')
+        )
+    }
+    environment {
+        tag = "$ref"
+        repository = "$repository"
+    }
+    stages {
+        stage('Download artifacts') {
+            agent {
+                docker {
+                    label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+                    image 'opensearchstaging/ci-runner:release-centos7-clients-v1'
+                    args '-e JAVA_HOME=/opt/java/openjdk-11'
+                    alwaysPull true
+                }
+            }
+            steps {
+                script {
+                    if ("$assets_url" != '') {
+                        withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', usernameVariable: 'GITHUB_USER', passwordVariable: 'GITHUB_TOKEN')]) {
+                            String assets = sh(
+                                script: "curl -H 'Accept: application/vnd.github+json' -H 'Authorization: Bearer ${GITHUB_TOKEN}' ${assets_url}",
+                                returnStdout: true
+                            )
+                            String assetUrl = null
+                            def parsedJson = readJSON text: assets
+                            def assetName = 'artifacts.tar.gz'
+                            parsedJson.each { item ->
+                                if (item.name == assetName) {
+                                    assetUrl = item.url
+                                }
+                            }
+                            echo "Downloading artifacts from $assetUrl"
+                            sh "curl -J -L -H 'Accept: application/octet-stream' -H 'Authorization: Bearer ${GITHUB_TOKEN}' ${assetUrl} -o artifacts.tar.gz && tar -xvf artifacts.tar.gz"
+                            stash includes: '**', name: 'odbc-release-artifacts'
+                        }
+                    }
+                }
+            }
+        }
+        stage('Release artifacts') {
+            parallel {
+                stage('Release Windows artifacts') {
+                    agent {
+                        docker {
+                            label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+                            image 'opensearchstaging/ci-runner:release-centos7-clients-v1'
+                            args '-e JAVA_HOME=/opt/java/openjdk-11'
+                            alwaysPull true
+                        }
+                    }
+                    steps {
+                        script {
+                                unstash 'odbc-release-artifacts'
+                                sh 'ls -l'
+                                publishToArtifactsProdBucket(
+                                    assumedRoleName: 'sql-odbc-upload-role',
+                                    source: "${WORKSPACE}/windows32-installer/OpenSearch-SQL-ODBC-Driver-32-bit-1.5.0.0-Windows.msi",
```

Comment:
Is it possible to avoid having hardcoded versions?

Reply:
Good catch. I was about to do that and forgot later. Let me see if its possible. I'll merge the PR by EOD today.

Reply:
@Yury-Fridlyand Will the tag always match the version here? For example in this case tag=1.5.0.0?

Reply:
Yes

Reply:
Fixed in 1197c1e39a16b3db0180f4f26108b62bcb7b47ff